packages/block-library/src/archives/index.php

```diff
@@ -17,11 +17,10 @@
 function render_block_core_archives( $attributes ) {
 	$show_post_count = ! empty( $attributes['showPostCounts'] );
 	$type            = isset( $attributes['type'] ) ? $attributes['type'] : 'monthly';
-	$class           = '';
```

Comment:
I think that initialization with an empty string is unnecessary since only a fixed string of `wp-block-archives-dropdown` or `wp-block-archives-list` will be set for `$class`.